docs/contributing/observability_developer_guide.md

```diff
@@ -185,7 +185,7 @@ implementations.
 
 <!-- Named Links -->
 
-[Events]: https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.16/#event-v1-core
+[Events]: https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.19/#event-v1-core
```

Comment:
```suggestion
[Events]: https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.21/#event-v1-core
```

Reply:
Ohh good call! Yes I will update to latest